pkgs/development/compilers/mrustc/patches/0001-use-shared-mrustc-minicargo.patch

```diff
@@ -0,0 +1,21 @@
+--- a/minicargo.mk
++++ b/minicargo.mk
+@@ -25,8 +25,8 @@
+ 
+ OUTDIR := output$(OUTDIR_SUF)/
+ 
+-MRUSTC := bin/mrustc
+-MINICARGO := tools/bin/minicargo
++MRUSTC ?= bin/mrustc
++MINICARGO ?= tools/bin/minicargo
```

Comment:
Does `=` work here?

Reply:
I use `?=` to set them from the environment to a real path, to limit the changes needed to the Makefile (they are both used as target names). If we want this patch to be upstreamed, `=` isn't an option.